### ISSUE TITLE
Interactive window does not render output Error loading preloads:

### ISSUE THREAD

**Issue by pedosb** [microsoft/vscode#132263]
- VSCode Version: 1.59.1
- Local OS Version: MacOS 11.5.2
- Remote OS Version: Ubuntu 20.04.2
- Remote Extension/Connection Type: Kubernetes
- Logs:
```
Error loading preloads:
    Failed to resolve module specifier 'https://vscode-remote%2Bk8s-container%2Bpodname%3Dvscode-server-%2Bnamespace%3Dai-ops-ns%2Bname%3Dvscode-server-epedbat%2Bimage%3D%252fai-ops-ns%252fmain%252fdocker-vscode%253alatest.vscode-resource.vscode-webview.net/home/nonroot/persistent-home/.vscode-server/extensions/ms-toolsai.jupyter-2021.8.1236758218/out/datascience-ui/errorRenderer/errorRenderer.js'
```

Steps to Reproduce:

1. Connect to the remote container using the "Attach Visual Studio Code" from the Kubernetes extension.
![Screen Shot 2021-09-02 at 10 47 06](https://user-images.githubusercontent.com/266610/131813358-42ab3dfe-721d-41d2-ab6d-5eee1c9096eb.png)
2. Start a Jupyter interactive session.
![Screen Shot 2021-09-02 at 10 49 24](https://user-images.githubusercontent.com/266610/131814346-b6b6d7a9-d65f-4aa8-b712-c262bed3a278.png)
3. Some outputs do not render, for example, errors.
![Screen Shot 2021-09-02 at 10 37 11](https://user-images.githubusercontent.com/266610/131813803-e63aa8e3-5198-44aa-abc6-a587d9baebc3.png)
4. I can see the output by changing the renderer to the built-in one
![Screen Shot 2021-09-02 at 10 37 31](https://user-images.githubusercontent.com/266610/131813996-26807b64-22a2-401a-8d02-8bcca1e58672.png)
![image](https://user-images.githubusercontent.com/266610/131814053-fe0d9301-7f0b-4caa-92e4-aaf90c23d267.png)
![Screen Shot 2021-09-02 at 10 37 44](https://user-images.githubusercontent.com/266610/131814075-cc4c7554-d8fa-4f6b-80bf-fe3bfbfa5651.png)

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: No

Does anyone know how to fix this? Or at least set the "built-in" renderer as the default one (in step 4 above), I have to click to change it every time the output does not render.

Thanks

**Comment by DonJayamanne** [comment]
@rebornix @roblourens /cc

**Comment by roblourens** [comment]
This may be fixed in 1.60, could you update and try there?

**Comment by pedosb** [comment]
Yes, it is fixed for me.

Thank you!
